tests/conftest.py

```diff
@@ -21,9 +21,14 @@ def execute(postgresql):
 
     loop = asyncio.new_event_loop()
 
-    def execute(query: str, commit: bool = False, autocommit: bool = False) -> None:
+    def execute(
+        query: str, commit: bool = False, autocommit: bool = False, dbname: str = None
```

Comment:
Should be `Optional[str] = None` (`Optional` to be imported from `typing`.)

Reply:
done in edc56aa